src/db/drizzle.config.ts

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from 'drizzle-kit'
+// biome-ignore lint/correctness/useImportExtensions: Drizzle can't find .js extension
+import { getConfig, isProduction } from '../infrastructure/config'
```

Comment:
Shouldn't the `db` folder be part of the infrastructure folder?

Reply:
Should it? 🤔 `prisma` isn't event in `src`. I added it as `src/db` because this is what Drizzle docs suggest, even though now that we've confirmed it works we could move it to a different level

Reply:
well, by DDD standards it i s infrastructure :-D

but following drizzle conventions may be less confusing for devs.

no strong opinion here

Reply:
I am not sure, to be honest, prisma `db-client` I think was living on `src` but it might not be true for this repo 😓

Reply:
No strong opinion either from my side, if we think it is better for DX, I am fine

Reply:
db-client was in the infrastructure folder, but prisma schemas on the root prisma